BorrowToOwn.API/Controllers/BrandController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using BorrowToOwn.Services.Communications.RequestObject.DTO;
+using BorrowToOwn.Services.Contracts;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BorrowToOwn.API.Controllers
+{
+    [ApiController]
+    [Route("/api/Brands")]
+    public class BrandController : Controller
+    {
+        private readonly IBrandService _brandService;
+
+        public BrandController(IBrandService brandService)
+        {
+            _brandService = brandService ?? throw new ArgumentNullException(nameof(brandService));
```

Comment:
is the null check necessary here? brandService can not be null if properly registered at the IoC level

Reply:
the thing is , if at any point lets say its not registered whether commented out or by any means , it will not just break the app. it will throw an exception

Reply:
well....